internal/runners/clean/run_win.go

```diff
@@ -4,34 +4,60 @@ package clean
 
 import (
 	"fmt"
+	"os"
 	"os/exec"
+	"path/filepath"
 
 	"github.com/gobuffalo/packr"
 
 	"github.com/ActiveState/cli/internal/language"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/osutils"
 	"github.com/ActiveState/cli/internal/scriptfile"
 )
 
-func removeConfig(cfg configurable) error {
-	return runScript("removeConfig", cfg.ConfigPath())
+func (u *Uninstall) runUninstall() error {
+	err := removeCache(u.cfg.CachePath())
+	if err != nil {
+		return err
+	}
+
+	err = removePaths(u.installPath, u.cfg.ConfigPath())
+	if err != nil {
+		return err
+	}
+
+	u.out.Print(locale.T("clean_success_message"))
+	return nil
 }
 
-func removeInstall(installPath string) error {
-	return runScript("removeInstall", installPath)
+func removeConfig(configPath string) error {
+	return removePaths(configPath)
 }
 
-func runScript(scriptName, path string) error {
+func removePaths(dirs ...string) error {
+	logging.Debug("Removing paths: %v", dirs)
+	scriptName := "removePaths"
 	box := packr.NewBox("../../../assets/scripts/")
 	scriptBlock := box.String(fmt.Sprintf("%s.bat", scriptName))
 	sf, err := scriptfile.New(language.Batch, scriptName, scriptBlock)
 	if err != nil {
-		return err
+		return locale.WrapError(err, "err_clean_script", "Could not create new scriptfile")
 	}
 
-	cmd := exec.Command("cmd.exe", "/C", sf.Filename(), path)
+	exe, err := os.Executable()
+	if err != nil {
+		return locale.WrapError(err, "err_clean_executable", "Could not get executable name")
+	}
+
+	args := []string{"/C", sf.Filename(), fmt.Sprintf("%d", os.Getpid()), filepath.Base(exe)}
```

Comment:
I suppose this only ensures that the `state.exe` stopped running, but the `state-tray` and `state-svc` executables may be running and hence cannot be removed...  But I guess that is a problem that I have to deal with in my PR. :)